### DISCUSSION TITLE
How to use data from fetching with the title in page source (Search Engine will able to Crawl)

### DISCUSSION THREAD

**Discussion by kittisakLS** [nuxt/nuxt#16671] [Questions]
I can't figure out how to waiting data from external api to use with `<Head>` or `useHead`
I already use `asyncData` and it works well in dev mode, but it does not work after build, It updates just in DOM but not in page source

**Comment by TheAlexLichter** [comment]
Ensure you do not have `ssr: false` and `await` the data in `<script setup>`.

For more help, please provide a [reproduction](https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction)

**Reply by kittisakLS** [reply]
Thank you for your answer.
It works in dev mode (localhost:3000), but in live mode. It does not wait the line of `await useFetch`, So in live mode my `console.log` always show null of data
